### PR TITLE
Olu teraform

### PR DESCRIPTION
**Pull request by olublessed**

### CODE REVIEW

**Review by justin-wilcoxen-reisys — COMMENTED** [review]
I understand that you are using the terraform modules with no extra "custom" modules.  But, from a sharing and reusability stand point, would it make sense to break out the AWS MySql script within the main.tf into its own tf file and maybe in directory.  That way if another ICF team were to come in and want to use the MYSql part of terraform they could easily find the script, rather than having to dig thru the main.tf.  Same kind of setup could go for the other AWS components too.  

Is this farfetched or crazy?  @jamiejackson and @olublessed

File: infrastructure/terraform/modules/ecs-cluster/user_data.sh [hunk 1]
```diff
@@ -96,6 +96,8 @@ if [ $? -ne 0 ]; then
 else
     echo "Directory $${EFS_MOUNT_DIR} is already a valid mountpoint!"
 fi
+#install mysql client to connect to RDS 
+yum -y install mysql
```
[the remark this answers is not in the document]
Comment (olublessed) [reply infrastructure/terraform/modules/ecs-cluster/user_data.sh]: mysql client is needed to run sql script from any of the nodes which is in the private subnets. Note that we are not starting the mysql server (service), we just need the client for connectivity to RDS instance.

**Comment by olublessed** [comment]
@jwilcoxen45 Your recommendation isn't farfetched, I will work on that in my next commit.

**Review by olublessed — COMMENTED** [review]
Removed the `yum -y install mysql` command from the userdata.sh
Will now merge this request to master

**Pull request merged**